torch/csrc/jit/runtime/symbolic_shape_registry.cpp

```diff
@@ -131,6 +131,62 @@ const std::string shape_compute_functions =
               out.append(li[i])
           return out
 
+        def index_select(self: List[int], dim: int, index: List[int]):
```

Comment:
nit?: should this have a return type `-> int`?

Reply:
return type gets inferred